.release-version

```diff
@@ -1 +1 @@
-5.4.0
+5.5.0-uat-testing-pagination
```

Comment:
probably want to revert this back to normal versioning, same with the workflow files, can remove the added lines to them

Reply:
Done